manifest.json

```diff
@@ -0,0 +1,144 @@
+{
+  "dependencies": {
```

Comment:
Ah, ok. Would it be difficult to start the migration in this PR, so that we can delete the corresponding stuff in install_third_party? We don't have to remove everything; I'm just looking for an intermediate state where a dependency is specified either here or there, but not both.